features/org.csstudio.dls.product.feature/rootfiles/css.sh

```diff
@@ -96,19 +103,24 @@ if [[ -n $opifile ]]; then
         fi
     fi
 
+    echo $macros
+    if [[ -n $macros ]]; then
+        macros_escaped=$(escape $macros)
+    else
+        macros_escaped=""
+    fi
+
     launch_opi_arg=--launcher.openFile
-    launch_opi="$opifile $macros"
+    launch_opi="$opifile $macros_escaped"
```

Comment:
It seems like the creation of the launch_opi string belongs at the end of the file, after the parsing of the links. This would place the code with similar purpose and structure together (parsing links and macros), and would prevent launch_opi being masked below as well.